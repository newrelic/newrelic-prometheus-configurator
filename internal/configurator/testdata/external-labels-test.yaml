common:
  external_labels:
    one: two
    three: four
  scrape_interval: 1m
  scrape_timeout: 1s

newrelic_remote_write:
  license_key: fake
