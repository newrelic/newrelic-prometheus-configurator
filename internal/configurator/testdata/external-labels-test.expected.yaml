global:
  external_labels:
    one: two
    three: four
  scrape_interval: 1m
  scrape_timeout: 1s
remote_write:
  - name: newrelic_rw
    url: "https://metric-api.newrelic.com/prometheus/v1/write"
    authorization: 
      credentials: fake
